doc/exclude_patterns.inc

```diff
@@ -5,12 +5,12 @@ modules/core/modules/otpinfo
 modules/core/modules/random
 modules/core/modules/trycatch
 modules/rtos/modules/FreeRTOS
-modules/inferencing
 modules/io/modules/xud
 modules/io/doc/substitutions.rst
 modules/sample_rate_conversion
-modules/voice/doc/substitutions.rst
+modules/lib_qspi_fast_read
```

Comment:
Why has the QSPI Fast Read documentation been excluded?

Reply:
It's not included in any toctree in this repo

Reply:
Thanks for the explanation.